src/src/components/uploads/editUploads.jsx

```diff
@@ -365,7 +365,7 @@ class EditUploads extends Component {
     // console.log("componentDidUpdate");
     // console.log(prevProps);
     // Typical usage (don't forget to compare props):
-    // console.debug(this.props.editingUpload.datasets);
+    // console.debug(this.props.editingUplo`ad.datasets);
```

Comment:
@BirdMachine can you fix this extra back quote? It's commented out but nice to get it right for now so it won't cause extra work when you enable that debugging line.